app/src/main/java/org/pesmypetcare/mypetcare/services/UserManagerService.java

```diff
@@ -43,16 +45,29 @@
 
     /**
      * Creates the user.
+     * @param username User username
      * @param uid User identifier
      * @param email User email
      * @param password User password
      */
-    void createUser(String uid, String email, String password);
+    void createUser(String uid, String username, String email, String password);
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5355440)